### DISCUSSION TITLE
Anyway to incorporate myMPD in order to stream https

### DISCUSSION THREAD

**Discussion by cmh714** [rern/rAudio#1339] [Feature Requests]
I have a stream or two doing https and mod wont handle that....Seems like this should work maybe?? https://jcorporation.github.io/myMPD/

Thanks All!

**Comment by ceejayemm** [comment]
rAudio plays https streams for me without any problem, eg:

https://listen-boomradio.sharp-stream.com/65_boom_radio_live_192

and the default Radio Paradise radios are https too.

Chris

**Reply by cmh714** [reply]
Thanks Chris and thats what I thought as this particular channel was working. Not sure what changed and why it doesnt work now....all good!
